packages/dd-trace/test/config.spec.js

```diff
@@ -761,6 +771,8 @@ describe('Config', () => {
     expect(config).to.have.nested.property('iast.maxContextOperations', 2)
     expect(config).to.have.nested.property('iast.deduplicationEnabled', true)
     expect(config).to.have.nested.property('iast.redactionEnabled', true)
+    expect(config).to.have.nested.property('iast.redactionNamePattern', null)
+    expect(config).to.have.nested.property('iast.redactionValuePattern', null)
```

Comment:
This test is `should give priority to the options`. It tests if the programmatic option is overriding the env var. This is not what you're doing here